### DISCUSSION TITLE
[Bug]Printer missing content

### DISCUSSION THREAD

**Discussion by Ackerven** [vuepress-theme-hope/vuepress-theme-hope#3109] [Q&A] [answered]
### Checklist

- [X] I am using **LTS** version of Node.js.
- [X] I am using the **latest v2 version** of VuePress and VuePress official plugins.
- [X] I am using the **latest v2 version** of vuepress-theme-hope and all plugins in this repo.
- [X] I followed the docs and I double checked my configuration.
- [X] I have searched [FAQ](https://theme-hope.vuejs.press/faq/) and I did not find the answer.

### Package name

vuepress-theme-hope

### Which operating system are you using?

- [ ] macOS
- [ ] Windows
- [X] Linux

### Describe the bug

When using the print function, the content wrapped in the custom container **details** (`::: details`) will be lost.

**Comment by Mister-Hope** [comment, marked as answer]
I think that's expected, as detail block is some further explanation, which could be omitted.
